build-tools/packages/build-cli/package.json

```diff
@@ -55,7 +55,7 @@
 		"postpack": "npm run clean:manifest",
 		"test": "npm run test:mocha",
 		"test:coverage": "c8 npm run test",
-		"test:mocha": "mocha --forbid-only \"test/**/*.test.ts\"",
+		"test:mocha": "mocha \"lib/test/**/*.test.*js\"",
```

Comment:
Our common mocharc file has [a feature](https://github.com/microsoft/FluidFramework/blob/920a65f66e0caad7e1b5e3df1e0afd3475a87c4a/packages/test/mocha-test-setup/mocharc-common.cjs#L82-L84) so we can make sure forbid-only is enabled in CI runs. But I noticed two things: a) build-cli doesn't depend on it so its mocharc doesn't extend it, like we do in other packages, and b) I'm not finding where we set FLUID_TEST_FORBID_ONLY for CI runs 🤔 . I'm pretty sure I've seen pipeline runs fail because they forgot to remove an only() but now I'd like to know where are we setting it.

So, I'd say either keep `--forbid-only` here, or further update to import mocha-test-setup and have the rc file import from the common one.

Reply:
I added forbid-only back. We use JSON config in build-cli and I don't want to convert it to CJS just so we can inherit
the common stuff. I still have dreams of [removing mocha entirely in build-tools](https://github.com/microsoft/FluidFramework/pull/21920).